module-2/backend/week_two.md

```diff
@@ -4,29 +4,51 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. At a high level, what is ActiveRecord? What does it do/allow you to do?
-2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
-3. What do they allow you to do?
-4. What's the difference between agile workflow and waterfall method?
-5. What is the difference between `#new` and `#create`?
-6. At a basic level, what does cURL allow you to do?
-7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
-8. Define foreign key, primary key, and schema.
-9. Describe the relationship between a foreign key on one table and a primary key on another table.
-10. What are the parts of an HTTP response?
-11. Describe some techniques to make our Sinatra code more DRY. Give an example of when you would use these techniques.
+1. At a high level, what is ActiveRecord? What does it do/allow you to do?  
+ActiveRecord is a Ruby gem that helps to simplify interactions with the database. It allows us to use keywords instead of building our own SQL query strings. Also helps us to set up and manage database table associations through short keywords.
+2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.  
```

Comment:
This is a stale version of the questions. Be sure to set the Turing repo up as an [upstream repository](https://help.github.com/articles/configuring-a-remote-for-a-fork/), and then [fetch the latest version](https://help.github.com/articles/syncing-a-fork/) before you start working on the questions for the week.